### DISCUSSION TITLE
Github Pages deployment fails with status code 502 and 404

### DISCUSSION THREAD

**Discussion by ms-studio** [community/community#22027] [Other Feature Feedback, Questions, & Ideas]
<p>Since today, all my Github Pages deployments fail for this repository:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/coursweb/feedback/actions">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/coursweb/feedback/actions" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/241637/181068341-2e6eb2c1-b239-44a9-9f47-2c1c6bad383a.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/05bc7ef74f02f0290813fa75d07682e663890868_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/5/05bc7ef74f02f0290813fa75d07682e663890868_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/5/05bc7ef74f02f0290813fa75d07682e663890868.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/05bc7ef74f02f0290813fa75d07682e663890868_2_10x10.png"></div>

<h3><a href="https://github.com/coursweb/feedback/actions" target="_blank" rel="noopener nofollow ugc">Actions · coursweb/feedback</a></h3>

  <p>Méthodes de feedback. Contribute to coursweb/feedback development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>The build process works fine, but deployment to github-pages results in an error.</p>
<p>At <a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a> there is currently no incident.</p>
<p>The error log looks like this:</p>
<pre><code class="lang-auto">Failed to create deployment for 0e626b47d0408d105e044de76c220c33b6c92eb2.
{"message":"Deployment request failed for 0e626b47d0408d105e044de76c220c33b6c92eb2 due to in progress deployment. 
Please cancel fda901df468880e913e3fbe5d659d1835353ae38 first or wait for it to complete.",
"documentation_url":"https://docs.github.com/rest/reference/repos#create-a-github-pages-deployment"}
Error: Error: Request failed with status code 400
</code></pre>
<p>Is there anything I can do? It doesn’t look to me like there is an “in progress deployment” going on – as far as I can see all previous deployments have a status of “canceled” or “failed”.</p>
<p>PS: it’s a small Jekyll website, the size of the “Artifact” after successful build is 640 KB. It doesn’t seem normal that the deployment to github-pages is taking 10+ minutes.</p>

**Comment by Disquietudee** [comment]
<p>They just updated the status to degraded performance.<br>
<a href="https://www.githubstatus.com/incidents/z95g34pl8rdm" rel="noopener nofollow ugc">GitHub Status - Incident with GitHub Pages</a></p>

**Comment by seancrain** [comment]
<p>I can confirm I am having the same issues with GitHub Pages. Deployments will either start running and get stuck in the updating_pages stage, or they will fail outright.</p>

**Comment by AndyDaMandy** [comment]
<p>I’m having a similar issue except my error code is 400. Perhaps it’s a github issue?</p>

**Comment by ms-studio** [comment, marked as answer]
<p>Indeed, it was a Github issue. Now everything is working again. :smiley:</p>
